src/odemis/acq/move.py

```
@@ -127,46 +135,44 @@ def __init__(self, microscope):
         pass
 
     @abstractmethod
-    def get_current_posture_label(self, pos: Dict[str, float] = None) -> int:
+    def get_current_posture(self, position: Dict[str, float] = None) -> Posture:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)